recipes/python-subunit/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "python-subunit" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python-subunit-{{ version }}.tar.gz
+  sha256: 042039928120fbf392e8c983d60f3d8ae1b88f90a9f8fd7188ddd9c26cad1e48
+
+build:
+  noarch: python
+  number: 0
+  script: "python -m pip install . -vv"
+  
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - extras
+    - testtools >=0.9.34
+    - six
+    
+test:
+  imports:
+    - subunit
+  commands:
+    - set
+
+about:
+  home: https://launchpad.net/subunit
+  license: Apache-2.0
+  license_file: COPYING
+  license_family: APACHE
```

Technically it's dual licensed (and we'll just skip license family  because of that):
```suggestion
  license: Apache-2.0 OR BSD-3-Clause
  license_file: COPYING
```